packages/kusama/src/__snapshots__/assetHubKusama.multisig.proxy.e2e.test.ts.snap

```diff
@@ -93,7 +93,7 @@ exports[`Kusama AssetHub Multisig Proxy > success tests > 2-of-3 multisig with p
     "data": {
       "approving": "D8ew585BL5H1ALhn4kmJoxhgqcgKDZLPc6xunJNv4mmrBns",
       "callHash": "(hash)",
-      "multisig": "HTcrVaUWS2Lz5V3cf7mQvvpLpqCRS4YpxLVDZWy67q2Rujp",
+      "multisig": "E9vsiNkTeYr8A5gt1k2udPrtaBy9KeQx8t3Ygp5Qy3MfQUu",
```

Comment:
is this expected?

Reply:
Yes, it's from #429, which changed the accounts' object used in multisig tests: https://github.com/open-web3-stack/polkadot-ecosystem-tests/pull/429/files#diff-b0f30b7bfd20a65bb56f31da000db90287c641d6b387c8520a0782affb839211L3